tests/assets/model/test_model.rad

```diff
@@ -4,65 +4,65 @@
 #   ============== MODIFIERS ==============
 
 
-void glass air_boundary
+void plastic DarkFloor
 0
 0
-3 1.0 1.0 1.0
+5 0.1 0.1 0.1 0.0 0.0
 
-void glass generic_interior_window_vis_0.88
+void glass generic_exterior_window_vis_0.64
 0
 0
-3 0.9584154328610596 0.9584154328610596 0.9584154328610596
+3 0.6975761815384331 0.6975761815384331 0.6975761815384331
 
-void plastic generic_interior_shade_0.50
+void glass CustomTriplePane
```

Comment:
?
```suggestion
void glass custom_triple_pane_0.3
```

Reply:
This is just a test file but, ok. I guess the tests sent an example for others.